ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityQuestion.component.ts

```diff
@@ -18,9 +18,11 @@ import { delay, filter, map, shareReplay, startWith, takeUntil, tap } from 'rxjs
                   [validationRequested]="validationRequested$ | async"
                   (valueChanged)="enteredValue$.next($event)">
           </ddp-activity-answer>
-          <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
-              <ddp-validation-message [message]="errorMsg">
-              </ddp-validation-message>
+          <div *ngIf="block.shown">
```

Comment:
The actual question itself checks `block.shown` in the `ActivityAnswerComponent`. So easiest fix seems to be adding a check here for the validation messages.